x-pack/plugins/synthetics/server/routes/monitor_cruds/get_monitor.ts

```diff
@@ -162,3 +175,64 @@ const parseLocationFilter = (serviceLocations: ServiceLocations, locations?: str
 export const findLocationItem = (query: string, locations: ServiceLocations) => {
   return locations.find(({ id, label }) => query === id || label === query);
 };
+
+export const getSyntheticsMonitorOverviewRoute: UMRestApiRouteFactory = () => ({
+  method: 'GET',
+  path: API_URLS.SYNTHETICS_OVERVIEW,
+  validate: {
+    query: querySchema,
+  },
+  handler: async ({ request, savedObjectsClient, server }): Promise<any> => {
+    const { perPage = 5 } = request.query;
+    const { saved_objects: monitors } = await getMonitors(
+      {
+        perPage: 1000,
+        sortField: 'name.keyword',
+        sortOrder: 'asc',
```

Comment:
The api will be adjusted to take sorting parameters as part of the sorting issue. Out of scope for this issue.